### ISSUE TITLE
/api/users/ gives 500 error due to duplicate leftOuterJoin

### ISSUE THREAD

**Issue by cxcorp** [TKOaly/user-service#121]
ER_NONUNIQ_TABLE: 'Not unique table/alias: 'payments'

from:
- https://github.com/TKOaly/tkoaly-admin-service/pull/1

```
2022-09-16T00:01:15.699Z knex:query select `users`.*, `payments`.`membership_applied_for`, max(`payments`.`valid_until`), `users`.* from `users` left outer join payments on (users.id = payments.payer_id and payments.valid_until > now()) left outer join payments on (users.id = payments.payer_id and payments.valid_until > now()) group by `users`.`id` order by `users`.`name` asc undefined

```

**Comment by cxcorp** [comment]
Caused by duplicate `leftOuterJoin()` in `UserDao.findAll()`;:
https://github.com/TKOaly/user-service/blob/b8e9cb78884208557c9608c83c3caa3ab32a0348/src/dao/UserDao.ts#L49-L65

Triggered by calling `/api/users/` with a token

**Comment by iritmaximus** [comment]
Seems like this was fixed in 5b672a6
